packages/rolldown/src/rolldown-build.ts

```diff
@@ -43,7 +42,3 @@ export class RolldownBuild {
     await this.#stopWorkers?.()
   }
 }
-
-function _assert() {
```

Comment:
This is used to ensure `RolldownBuild` has properies `generate` and `write` by `tsc`.

Reply:
Thanks for the reply, I didn't understand it before.